src/exceptions/JobException.class.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Tripod\Exceptions;
+
+// @codeCoverageIgnoreStart
+require_once TRIPOD_DIR . '/exceptions/Exception.class.php';
```

Comment:
I assume this was added out of necessity, can you explain what was going :boom: to add this?  I've been trying to move Tripod away from these explicitly require_once's